packages/contracts/test/transactionManager.spec.ts

```diff
@@ -240,9 +244,21 @@ describe("TransactionManager", function () {
     const receipt = await prepareTx.wait();
     expect(receipt.status).to.be.eq(1);
 
+    const variantDigest = await getVariantTransactionDigest({
+      amount: record.amount,
+      expiry: record.expiry,
+      preparedBlockNumber: receipt.blockNumber,
+    });
+
+    expect(await instance.variantTransactionData(invariantDigest)).to.be.eq(variantDigest);
+    // expect(await instance.activeTransactionBlocks(preparer.address));
+
+    // const activeBlock = await instance.activeTransactionBlocks(transaction.user, 0);
```

Comment:
We should return these assertions, they are necessary

Reply:
We'll likely have to account for the following cases:
1. multiple txs are prepared in the same block for the same user
2. user has existing prepared txs

Reply:
> multiple txs are prepared in the same block for the same user
we can test this out to check the activeTransactionBlocks>

> user has existing prepared txs
already taking care of this in multiple test cases

> assert the liquidity changes/lockup
> assert no additional funds were sent to the contract (i.e. finalContractBalance == initialContractBalance

will add these